worldengine/draw.py

```diff
@@ -552,7 +517,7 @@ def draw_precipitation(world, target, black_and_white=False):
         low = world.precipitation['data'].min()
         high = world.precipitation['data'].max()
         floor = 0
-        ceiling = 255
+        ceiling = 255  # could be changed into 16 Bit grayscale easily
```

Comment:
The used image-writer has a function to retrieve the maximum color-depth (```get_max_colors()```, I believe); but since this function could be called by UI-functions they would have to provide a function like that, too. (Although it'll always return 255 in that case, I assume.) So I didn't touch the constant value for now.